pages/common/vsce.md

```diff
@@ -0,0 +1,23 @@
+# vsce
+
+> Extension manager for Visual Studio Code.
+
+- Publish an extension as major, minor or patch version:
+
+`vsce publish {{version}}`
```

Comment:
There is a `sub-command` to give access to other users, but not (like in `npm`)  to login yourself. `vsce` asks for a personal access token (PAT) on first use. Alternatively, it can be specified for each operation by specifying the `--pat` flag.